kickstarts/base.ks.erb

```diff
@@ -73,6 +73,12 @@ do
   grep -q 'DEVICE=' $FILE || echo "DEVICE=$DEVICE" >> $FILE
 done
 
+# Disable zeroconfig to allow access to meta-data service by cloud-init
+cat > /etc/sysconfig/network << EOF
```

Comment:
There is nothing there and this is what was done to fix this in the fedora kickstarts [1]

[1] https://lists.fedoraproject.org/pipermail/cloud/2013-June/002504.html

That said it might be safest going forward to append... and it couldn't hurt.
I'll do that.
Thank you @abellotti !